packages/flutter/lib/src/services/app_messages.dart

```diff
@@ -0,0 +1,64 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter/shell.dart';
+import 'package:mojo/core.dart' as core;
+import 'package:sky_services/flutter/platform/app_messages.mojom.dart';
+
+/// APIs for exchanging messages with the host application.
```

Comment:
This doesn't look like it's intended to be a dartdoc.  It should probably be `//` rather than `///`